packages/web/app/src/lib/preflight-sandbox/allowed-globals.ts

```diff
@@ -0,0 +1,61 @@
+// initial list comes from https://github.com/postmanlabs/uniscope/blob/develop/lib/allowed-globals.js
+export const ALLOWED_GLOBALS = [
```

Comment:
I don't think someone needs `Buffer` or even `Function`. Not sure if it is a requirement for the crypto lib we added. If it's not needed let's cut down on the globals

Reply:
removed all global variables which ends with `Buffer` and `Array`